### DISCUSSION TITLE
Can I change my twitter field with a fedivese field on my profile?

### DISCUSSION THREAD

**Discussion by bogomil** [community/community#38560] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear Github,
I have a favor to ask. I want to add my fediverse (a.k.a Mastodon) account to my GitHub profile, but I need help. Could you please give equal opportunities to everyone to share their “social” accounts with everyone? It’s unfair to offer just a Twitter account.

Thanks,
Bogo, a GitHub citizen since 2010

![](https://talkweb.eu/wp-content/uploads/2022/11/twitter_gh.png)

**Comment by peaz** [comment]
Following and upvoting this. Easily fixed by making it a generic social platform URL field.

**Comment by bogomil** [comment]
I am not sure whether this is the correct category, but let's see :)

**Comment by david-engelmann** [comment, marked as answer]
[Duplicate of this discussion](https://github.com/community/community/discussions/5720)

**Reply by bogomil** [reply]
Thanks!

**Comment by derekprior** [comment]
Hey folks, I just wanted to give you a heads up that we are working on this. It'll ship when it's ready, but with holiday season vacations coming up that won't be for a bit yet. Thanks for your patience.

**Reply by bogomil** [reply]
Thanks @derekprior. Is the plan to add a fediverse field or make it more customizable, so anyone can add whatever they see fit?

**Reply by derekprior** [reply]
Right now, the plan is to allow a limited number of links to sites of your choosing with some additional UI support for recognized links.
